### PR TITLE
Processes withdrawn event

### PR DESCRIPTION
**Pull request by tnickelsen**

### CODE REVIEW

File: src/ProjectOrigin.Chronicler.Test/ChroniclerRepositoryTests.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 using AutoFixture;
 using Dapper;
 using FluentAssertions;
+using GraphQL.Types.Relay.DataObjects;
```
Comment (MartinSchmidt) [inline src/ProjectOrigin.Chronicler.Test/ChroniclerRepositoryTests.cs:7]: ? hvad er det

**Comment by MartinSchmidt** [comment]
@tnickelsen  You are missing a bit of test coverage https://sonarcloud.io/component_measures?id=project-origin_chronicler&pullRequest=6&metric=new_coverage&view=list

File: src/ProjectOrigin.Chronicler.Test/ChroniclerRepositoryTests.cs (hunk 1 above)
Comment (tnickelsen) [reply src/ProjectOrigin.Chronicler.Test/ChroniclerRepositoryTests.cs:7]: fjernet

**Review by MartinSchmidt — APPROVED** [review]

**Comment by MartinSchmidt** [comment]
@tnickelsen  Looks good to me, I have created a new issue #7 for if the other certificate is withdrawn, then we don't currently corrrect the statistic.

**Pull request merged**